tests/expect/composer_paths_08.exp

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env -S expect -f
+
+set timeout 3
+spawn ../../bin/composer_paths.sh "" "../fixtures/out-of-sync-lock-file"
```

Comment:
It's failing because of a silly typo. 😄 

You have `out-of-sync-lock-file` as the directory name throughout this test, but the actual directory name is `out-of-sync-lock` (no `-file` on the end).

Reply:
Argh! You know that feeling when you've stared yourself blind on something and couldn't see it. That.

Fixed now, but the tests is still failing... 😕